src/Storages/MergeTree/MergeTreeRangeReader.cpp

```diff
@@ -859,22 +859,28 @@ static ColumnPtr combineFilters(ColumnPtr first, ColumnPtr second)
         return first;
     }
 
-    auto mut_first = IColumn::mutate(std::move(first));
-    FilterDescription firsrt_descr(*mut_first);
+    FilterDescription firsrt_descr(*first);
```

Comment:
@KochetovNicolai please change name `firsrt_descr` to something readable.

Reply:
ah, yes